src/EditorFeatures/Core.Wpf/Preview/PreviewFactoryService.cs

```diff
@@ -253,12 +254,42 @@ private bool ProjectReferencesChanged(ProjectChanges projectChanges)
             return false;
         }
 
-        public Task<object> CreateAddedDocumentPreviewViewAsync(Document document, CancellationToken cancellationToken)
+        async Task<object> IPreviewFactoryService.CreateAddedDocumentPreviewViewAsync(Document document, CancellationToken cancellationToken)
```

Comment:
What's using these that we can't just delete them?

Reply:
Only one of these was used, and only in a test. These are now removed.